stock_request/models/stock_request.py

```diff
@@ -177,6 +177,14 @@ def _compute_qty(self):
                 request.product_id.uom_id._compute_quantity(
                     open_qty, request.product_uom_id)
 
+    def get_parents(self):
+        location = self.location_id.sudo()
```

Comment:
Why this?

Use of sudo is to be confidential. Remember you switch to new environment.

In which case sudo is mandatory ?

Reply:
We search with sudo because the user may not have permissions to view the locations, but we must access them in order to obtain routes as it is used on https://github.com/OCA/stock-logistics-warehouse/pull/392/commits/262672cb4f223c54d853a93e4a9868989e5eeb5e#diff-cb7ac1537b7c039cfc237796ef052a90L212